api/migrations/20170509230917_add_email_id_column.js

```diff
@@ -1,20 +1,8 @@
 
 exports.up = function(knex, Promise) {
     return knex.schema.table('SIP_user', table => {
-      table.string('email_id').notNullable().unique();
-    }).then(knex.select().from('SIP_user').then(users => {
-      if (process.env.OIDC_DB_ADAPTER === 'mysql') {
-        return knex.raw(`
-          UPDATE SIP_user SET email_id=LCASE(email);
-        `);
-      } else {
-        users.forEach(user => {
-          const emailID = user.email.toLowerCase();
-          knex('SIP_user').update('email_id', emailID).where('id', user.id);
-        });
-
-      }
-  }));
+        return table.string('email_id').notNull();

```

Comment:
The email id was discussed with Bob in order to fix the case sensitive issue on Tathata. I have added the migration for existing users back in this commit. Removed it because everything I was working on had users coming in through the seed, but this will still be needed. The seeds will have to be updated in Tathata and Strata.